packages/core/src/background/handlers/Tabs.ts

```diff
@@ -127,7 +131,13 @@ export default class Tabs {
   }
 
   // eslint-disable-next-line @typescript-eslint/require-await
-  public async handle<TMessageType extends PolyMessageTypes> (id: string, type: TMessageType, request: PolyRequestTypes[TMessageType], url: string, port: chrome.runtime.Port): Promise<PolyResponseTypes[keyof PolyResponseTypes]> {
+  public async _handle<TMessageType extends PolyMessageTypes> (id: string, type: TMessageType, request: PolyRequestTypes[TMessageType], url: string, port: chrome.runtime.Port): Promise<PolyResponseTypes[keyof PolyResponseTypes]> {
+    this.#state.ensureUrlAuthorized(url);
+
+    if (type !== 'pub(authorize.tab)') {
+      this.#state.ensureUrlAuthorized(url);
```

Comment:
The whole point of this PR is to make this method call possible. This is how to check whether an app is authorized before answering its requests.

Reply:
Previously, applications could access certain APIs without prior authorization. To test, run this code in chrome browser console, on any App URL that is NOT authorized yet. After this PR, the wallet should reject the request, but on `develop`, wallet would respond fine :/.

```
var msg = {"id":"0.1","message":"","origin":"page","request":{"origin":"Some app"}};
msg.message = "pub(accounts.list)";
window.postMessage(msg, "*");
```